eng/testing/tests.singlefile.targets

```diff
@@ -103,6 +107,79 @@
     </ItemGroup>
   </Target>
 
+  <Target Name="ExcludeExistingR2RBinaries"
```

Comment:
Agree with @ViktorHofer here - copies like this can cause extra work and even create incremental build issues when they use wildcards like this.  It'd be good to understand them.